GDJS/Runtime/events-tools/commontools.js

```diff
@@ -31,6 +32,40 @@ gdjs.evtTools.common.getVariableString = function (variable) {
   return variable.getAsString();
 };
 
+/**
+ * Compares the boolean value of a variable. Equivalent to `variable.getAsBoolean()`.
+ * This shorcut function is needed for events code generation.
+ *
+ * @param {gdjs.Variable} variable
+ * @param {boolean} compareWith
+ * @returns {boolean} The content of the variable, as a string.
+ * @private
+ */
+gdjs.evtTools.common.getVariableBoolean = function (variable, compareWith) {
+  return variable.getAsBoolean() === compareWith;
+};
+
+/**
+ * Set the boolean value of a variable. Equivalent to `variable.setBoolean()`.
+ * This shorcut function is needed for events code generation.
+ *
+ * @param {gdjs.Variable} variable
+ * @param {boolean} bool The new boolean value of the variable.
+ * @private
+ */
+gdjs.evtTools.common.setVariableBoolean = function (variable, bool) {
```

Comment:
Let's rename `bool` to `value`, to avoid any clash with a reserved keyword in the future in case of porting this to another language.